media-libs/openimageio/metadata.xml

```diff
@@ -7,9 +7,12 @@
 	</maintainer>
 	<use>
 		<flag name="dicom">Enable DICOM medical image file support via <pkg>sci-libs/dcmtk</pkg></flag>
+		<flag name="iv">Enable iv (image viewer) tool</flag>
```

Comment:
Are you okay with the `tools` useflag? Building the tools wasn't optional before.

My thought was to make it more obvious that the `qt{5,6}` flags builds only iv. But I don't care either way, so I will revert `iv` to `qt5`.

Reply:
> Are you okay with the `tools` useflag? Building the tools wasn't optional before.

I think `tools` is fine since it is enabled by default. So introducing that flag shouldn't cause any surprises on upgrade.

> My thought was to make it more obvious that the qt{5,6} flags builds only iv. But I don't care either way, so I will revert iv to qt5.

`iv` does indeed make it more obvious what the flag does. However, I don't think the increase in clarity ways up against the cost of end users having to reconfigure and re-emerge. 

Unfortunately sometimes we are stuck with poor use flag names because the principle of least surprise tells us that changing the name of the flag is a bad idea. For the renaming of packages we have mechanisms in place to make this possible without inconveniencing the end user too much, there is currently no such mechanism for use flags unfortunately.